Competition2018/src/main/java/competition/ElectricalContract2018.java

```diff
@@ -74,4 +74,6 @@ public DeviceInfo(int channel, boolean inverted) {
     public abstract boolean climbReady();
 
     public abstract DeviceInfo getClimbMaster();
+    
+    public abstract DeviceInfo getPaulSolenoid();
```

Comment:
https://en.wikipedia.org/wiki/Ratchet_(device)

A ratchet consists of a round gear (see Figure 1) or linear rack with teeth, and a pivoting, spring-loaded finger called a pawl (or click[1][2]) that engages the teeth